packages/chatbot-server-mongodb-public/src/processors/makeStepBackUserQuery.eval.ts

```diff
@@ -0,0 +1,200 @@
+import { Scorer, EmbeddingSimilarity } from "autoevals";
+import { Eval } from "braintrust";
+import {
+  makeStepBackUserQuery,
+  StepBackUserQueryMongoDbFunction,
+} from "./makeStepBackUserQuery";
+import { AzureOpenAI } from "openai";
+import { Message, ObjectId, updateFrontMatter } from "mongodb-chatbot-server";
+import { assertEnvVars } from "mongodb-chatbot-server";
+import { evalEnvVars } from "../evalEnvVars";
+import { MongoDbTag } from "../mongoDbMetadata";
+
+const {
+  JUDGE_OPENAI_API_KEY,
+  JUDGE_EMBEDDING_MODEL,
+  OPENAI_API_KEY,
+  OPENAI_PREPROCESSOR_CHAT_COMPLETION_DEPLOYMENT,
+  OPENAI_ENDPOINT,
+  OPENAI_API_VERSION,
+} = assertEnvVars({
+  ...evalEnvVars,
+  OPENAI_PREPROCESSOR_CHAT_COMPLETION_DEPLOYMENT: "",
+  OPENAI_API_KEY: "",
+  OPENAI_ENDPOINT: "",
+  OPENAI_API_VERSION: "",
+});
```

Comment:
Would be nice to pull these vars and/or their derived objects (e.g. `openAiClient`) from the config file

Reply:
created new `evalHelpers` file for this